### ISSUE TITLE
[Bug Report][3.3.6] Expansion panel closing animation is jumping

### ISSUE THREAD

**Issue by Bjornar97** [vuetifyjs/vuetify#17741]
### Environment
**Vuetify Version:** 3.3.6
**Last working version:** 3.1.7
**Vue Version:** 3.3.4
**Browsers:** Chrome 114.0.0.0
**OS:** Windows 10

### Steps to reproduce
1. Put in a v-expansion-panels, and then put 1 v-expansion-panel inside it. 
2. Open and close the expansion panel

### Expected Behavior
The animation to open and close is smooth.

### Actual Behavior
The open animation is smooth, but the closing animation is jumpy.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqlUT1v20AM/SvMLW4BSxpSdDCcIkVRoEPHAh2iDLKOTtneV46Um8Lwfy9PshHBzhAgi07Hx3uP7/Fubz6nVO8GNCuzFvTJdYKf2gCw3lVdSuPveMGn1AWmGCo90fEReQk7IQBC4vCmNT/K2ZoZgE+i9e8xowdKPHiw0cUMTAKdR4E+BsZeUIYMnaVETD2FB0BHUsOX6H20tITcieoiWNyQEMPjoB9kAddtlBx20Q1JtGkz8BV87R4HhH5IZEmLCJ4C+e55smdXzYWtUxaXyBRGAabI1s0sSr1ynykJsJqZEiWfYhbYQ8YtHGCbo4eFbmFR2mH0LuD5AW5Kx7vFN3Quws+Ynb1avC8CE6WSmaWZ2Crfpfo3x6Cr3BeW9ghwa1YwVkpNVcq9Nb9EEq+aprdBn1nNdZfrgNKE5JtbbWvyEIQ8Vjb62+v6uv7QWGKZl2tkX21y/MuYlaQ1y5mM0PbfK6SOnaPCx0nhWKp0h1wkLqgb1dthrjIGixnzax2dPZu7OoMunBX1QxsOmrej8IfPou55jPnuNOSbLI9khem+CB6WZpxTlccH5v4/DcVCGg==)

### Other comments
This bug is also on the documentation page for Expansion panels. In the reproduction link, I just copied the example from the documentation.

**Comment by KaelWD** [comment]
When you say "jumpy" do you mean like #15523?

**Comment by Bjornar97** [comment]
I completely missed that you had commented, sorry for late response.

No, its in the middle of the animation, when it feels like it stops the animation and it jumps to end state immediately, or something like that, 

I made a screen recording showing the difference between 3.1.7 and 3.3.6.

Jumpy animation, version 3.3.6
https://github.com/vuetifyjs/vuetify/assets/1843267/5107717a-5a5b-42bb-8b06-bad91eba5057

Smooth animation, version 3.1.7
https://github.com/vuetifyjs/vuetify/assets/1843267/9e71269a-0583-4c1d-a40e-13c8a31e85a1


I did however find something interresting. Im working from home today, and on my home computer the animation is smooth and there is no issue. My home computer runs windows 11, but my work computer (where the issue is) runs windows 10. I tested in chrome on both machines.
